### ISSUE TITLE
Add share button near play and pause

### ISSUE THREAD

**Issue by lm-n** [processing/p5.js-web-editor#1449]
#### Nature of issue
- Existing feature enhancement

#### Feature enhancement details:
Make sharing easier and faster by adding a share button in the GUI near play and pause buttons. This button would take users to the current share interface without having to navigate the file menu.

**Comment by lm-n** [comment]
There's a share icon already in the editor ![Share Icon](https://user-images.githubusercontent.com/16418450/85082787-30799180-b18d-11ea-947b-3940efc98e15.png). Perhaps a button with that icon could be placed between the pencil of the project name and the author's name.

**Comment by saijatin28** [comment]
Hey! @catarak . What do you think about placing ![](https://user-images.githubusercontent.com/16418450/85082787-30799180-b18d-11ea-947b-3940efc98e15.png) **in between** the pause button and the auto refresh check box.
![image](https://user-images.githubusercontent.com/39066840/86400253-e84a8b00-bcc5-11ea-9f6e-2d900bfc7270.png)


Upon clicking this icon :
1. We could have the link to the collection copied to the clipboard
2. Display a toast stating the same.


Also, as a side note, currently when we click **Share** we are just given the link and have to manually copy it. I think that copying the link to the clipboard and displaying a toast message would make more sense. Should I open another feature request for the same ?
![image](https://user-images.githubusercontent.com/39066840/86400682-b2f26d00-bcc6-11ea-9ff5-70ce56f62d9d.png)

**Comment by catarak** [comment]
> Also, as a side note, currently when we click **Share** we are just given the link and have to manually copy it. I think that copying the link to the clipboard and displaying a toast message would make more sense. Should I open another feature request for the same ?

For me, when I click on the input box, it copies it to my clipboard, and a message appears that says "Copied to Clipboard". Does that not happen for you? If so, that is a bug.

**Comment by saijatin28** [comment]
> For me, when I click on the input box, it copies it to my clipboard, and a message appears that says "Copied to Clipboard". Does that not happen for you? If so, that is a bug.

No! You're right @catarak . I just realised that I had to click the input box for it to copy it to the clipboard.
Earlier I was expecting it to copy to the clipboard when I clicked the share button.

**Comment by catarak** [comment]
No worries!!
